src/App.jsx

```diff
@@ -1,17 +1,40 @@
+import { useState } from 'react';
 import './App.css';
+import messages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [chatMessages, setChatMessages] = useState(messages);
+
+  const handleLikeToggle = (id) => {
+    setChatMessages((prevMessages) =>
+      prevMessages.map((message) =>
+        message.id === id
+          ? { ...message, liked: !message.liked }
+          : message
+      )
+    );
+  };
+
+  const totalLikes = chatMessages.filter((message) => message.liked).length;
+  const person1 = messages[0].sender;
+  const person2 = messages[1].sender;
```

Comment:
What happens here if the first two messages from are texts from the same person?